### ISSUE TITLE
ncdu: Copy Path to Clipboard

### ISSUE THREAD

**Issue by gary-kim** [ncw/rclone#3265]
#### What is your current rclone version (output from `rclone version`)?
```
rclone v1.48.0
- os/arch: linux/amd64
- go version: go1.12.6
```
#### What problem are you are trying to solve?

`ncdu` is really nice for browsing files on a remote but to perform operations on a file or folder found through `ncdu` you have to remember the path.

#### How do you think rclone should be changed to solve that?

Having a key to copy the path of the currently selected file or folder to the clipboard would make it easy to just paste the path to be able to perform operations.

If the idea is okay, I'll go ahead with a pull request.

**Comment by ncw** [comment]
Sounds like a great idea - please go ahead :-)

Not sure how easy it will be on 3 different platforms so maybe just displaying the full path on a key might be enough?

**Comment by gary-kim** [comment]
Displaying the path would be nice. As far as actually putting on the clipboard, `github.com/atotto/clipboard` seems to support the 3 main platforms. Would that work?

**Comment by ncw** [comment]
> Displaying the path would be nice. As far as actually putting on the clipboard, `github.com/atotto/clipboard` seems to support the 3 main platforms. Would that work?

That looks like a small focussed library, so yes I think that would be fine.

We probably need to be a little careful with error handling as there may be times the clipboard doesn't work (eg running via ssh) or on plan9 etc, but yes, let's go for it!

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.49

Thank you :-)
